src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -469,6 +471,8 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
     manager.register_pass<pass::BlockLSTMReplacer>();
     manager.register_pass<pass::GRUBlockCellReplacer>();
     manager.register_pass<pass::ConstToResultRemover>();
+    manager.register_pass<pass::SwitchMergeResolver>();
+    manager.register_pass<ov::pass::UnrollIf>();
```

Comment:
minor: ov::pass -> pass::

Reply:
Now it will be conflicting with passes from `ov::frontend::tensorflow::pass` namespace so we need to list the full namespace for these particular passed.